### PR TITLE
TBB suppressions for bug #5075 added to the valgrind.supp file

### PR DESCRIPTION
**Pull request by kiran4399**
Bug #5075. TBB suppressions added

### CODE REVIEW

**Comment by alalek** [comment]
`data/` directory is a bad place for build-related files. Please move it into `platforms/linux`

File: data/valgrind_sup/memleak.supp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+   fun:_Znam
+   obj:/usr/lib/libtbb.so.2
+   obj:/usr/lib/libtbb.so.2
+   obj:/usr/lib/libtbb.so.2
+   obj:/usr/lib/libtbb.so.2
+   fun:_ZNK3tbb8internal32allocate_root_with_context_proxy8allocateEm
+   fun:_ZN2cv13parallel_for_ERKNS_5RangeERKNS_16ParallelLoopBodyEd
+   fun:_ZN2cv12CvtColorLoopINS_8RGB2GrayIhEEEEvRKNS_3MatERS3_RKT_
+   fun:_ZN2cv8cvtColorERKNS_11_InputArrayERKNS_12_OutputArrayEii
+   fun:main
```
Comment (alalek) [inline data/valgrind_sup/memleak.supp]: Actually, last 3 functions are useless here

**Comment by mshabunin** [comment]
Actually, valgrind suppression file already exists: 
- https://github.com/Itseez/opencv/blob/master/platforms/scripts/valgrind.supp
- https://github.com/Itseez/opencv/blob/2.4/platforms/scripts/valgrind.supp

Please, add your suppressions there.

File: platforms/scripts/valgrind.supp [hunk 2]
```diff
@@ -4,3 +4,29 @@
...
+   fun:_Znam
+   obj:/usr/lib/libtbb.so.2
+   obj:/usr/lib/libtbb.so.2
+   obj:/usr/lib/libtbb.so.2
+   obj:/usr/lib/libtbb.so.2
+   fun:_ZNK3tbb8internal32allocate_root_with_context_proxy8allocateEm
+   fun:_ZN2cv13parallel_for_ERKNS_5RangeERKNS_16ParallelLoopBodyEd
+   fun:_ZN2cv12CvtColorLoopINS_8RGB2GrayIhEEEEvRKNS_3MatERS3_RKT_
+   fun:_ZN2cv8cvtColorERKNS_11_InputArrayERKNS_12_OutputArrayEii
+   fun:main
```
Comment (alalek) [inline platforms/scripts/valgrind.supp]: Problem with useless "main+cvtColor" filtering is still here. Please truncate stack trace before `parallel_for_` call (see `fun:ippicvStaticInit` sample above).

Also please squash commits into one (to remove unnecessary intermediate changes from Git history)

**Comment by vpisarev** [comment]
this PR will be recreated because of CI issues:

**Pull request closed without merging**